Service/RequestService.php

```diff
@@ -497,6 +497,22 @@ public function requestHandler(array $data, array $configuration): Response
                 } else {
                     //$this->data['query']['_schema'] = $this->data['endpoint']->getEntities()->first()->getReference();
                     $result = $this->cacheService->searchObjects(null, $filters, $allowedSchemas['id']);
+
+                    if (count($allowedSchemas['id']) === 1) {
```

Comment:
Why do we only want to do this if we have 1 allowedSchema ?
Because this is only for the normal/CRUD 'GET Collection' calls ?